mifospay/src/main/res/values/strings.xml

```diff
@@ -132,11 +132,11 @@
     <string name="link_your_bank_account">Link Your Bank Account</string>
     <string name="no_linked_bank_accounts">No Linked Bank Accounts</string>
     <string name="editable_image_hint">Editable image hint</string>
-    <string name="change_password">Change Password</string>
+    <string name="change_password">Changes your Account Password</string>
     <string name="current_password">Current Password</string>
     <string name="new_password">New Password</string>
     <string name="confirm_new_password">Confirm New Password</string>
-    <string name="change_passcode">Change Passcode</string>
+    <string name="change_passcode">Changes your Account Passcode</string>
```

Comment:
I think it should be `change` not `changes`

Reply:
Okay, will change it

Reply:
Okay, will change it